usaspending_api/download/helpers/download_annotation_functions.py

```diff
@@ -25,6 +28,7 @@ def universal_award_matview_annotations():
         "federal_accounts_funding_this_award": StringAgg(
             "award__financial_set__treasury_account__federal_account__federal_account_code", ";", distinct=True
         ),
+        "usaspending_permalink": Concat(Value(HOST + "/#/award/"), "award__generated_unique_award_id"),
```

Comment:
Unfortunate question: do we need to url-encode the GUAI? (I believe yes. cc @emily-brents)
Next unfortunate question: how do we do this in SQL?

I found [this extension](https://github.com/okbob/url_encode) but it isn't supported in AWS RDS.
See [SO](https://stackoverflow.com/questions/10318014/javascript-encodeuri-like-function-in-postgresql)

Reply:
Certain special characters have to be URL encoded for API calls, but the frontend currently results in a 404 for most special characters. There is an issue in Sprint 97 for the frontend special characters that hasn't been fixed yet -  https://federal-spending-transparency.atlassian.net/browse/DEV-3994. I would assume that the fix for that involves URL encoding the GUAI, but can't say for sure.

Reply:
Choosing to defer this unfortunate "edge case" for future work.